tools/autograd/derivatives.yaml

```diff
@@ -110,6 +110,25 @@
 #     destroy saved buffers if we know variables are not going to be retained,
 #     e.g., it is used by _cudnn_rnn
 #
+# In a gradient expression, the following functions are in scope:
```

Comment:
we don't need this additional header here right? it kind of sounds like this is the only function in scope in the gradient expression.

Reply:
This is true, but the section above is about variables. I could move it there while replacing `variables` with `variables/functions`.

Reply:
Ahh mb didn't realize the difference, combining sounds good to me!